CRM/Contact/BAO/SavedSearch.php

```diff
@@ -87,11 +87,13 @@ public static function retrieve(&$params, &$defaults) {
    *
    * @param int $id
    *   The id of the saved search.
+   * @param bool $isCustomSearch
+   *   Are you fetching formValues for a custom search form?
    *
    * @return array
    *   the values of the posted saved search used as default values in various Search Form
    */
-  public static function getFormValues($id) {
+  public static function getFormValues($id, $isCustomSearch = FALSE) {
```

Comment:
Thanks @monishdeb - just at the very trivial level - given all the other checks you have - do we need this parameter - it seems that the method_exists & the customSearchClass check should be enough